### PR TITLE
[ja] Localize Infrastructure as Code (IaC) into Japanese

### LINKED ISSUE #2856: [ja] Localize Infrastructure as Code (IaC) into Japanese
Localize https://glossary.cncf.io/infrastructure-as-code/ into Japanese

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Infrastructure as Code (IaC) into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2856 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview:

- https://deploy-preview-2892--cncfglossary.netlify.app/ja/infrastructure-as-code/

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/infrastructure-as-code.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+Infrastructure as Codeは、インフラストラクチャーの定義を一つあるいは複数のファイルで保存する実践を指します。
+これは、通常はシェルスクリプトや他の設定ツールを用いたInfrastructure as a Serviceを手動でプロビジョニングする従来のモデルに代わるものです。
+
+## 解決すべき問題はなんですか
+
+クラウドネイティブな方法でアプリケーションを構築するには、インフラストラクチャーを使い捨てできるようにし、かつ再現可能にする必要があります。
+また自動化された繰り返し可能な方法で、人の手が介入することなくオンデマンドに[スケール](/ja/scalability/)する必要があります。
+手動でのプロビジョニングは、クラウドネイティブアプリケーションの応答性とスケーラビリティを満たすことができません。
```
Comment (kaitoii11) [inline content/ja/infrastructure-as-code.md:15]: クラウドネイティブアプリケーションにリンクの追加お願いします。

Comment (Okabe-Junya) [reply content/ja/infrastructure-as-code.md]: ありがとうございます、

https://github.com/cncf/glossary/pull/2892/commits/8adb62603a1f52639d9c483ad538767bba968b85 で修正しました。

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**